server/webserver.go

```diff
@@ -90,53 +90,51 @@ func (s *LivepeerServer) cliWebServerHandlers(bindAddr string) *http.ServeMux {
 		}
 
 		pricePerUnit := r.FormValue("maxPricePerUnit")
-		pr, err := strconv.ParseInt(pricePerUnit, 10, 64)
-		if err != nil {
-			glog.Errorf("Error converting string to int64: %v\n", err)
-			return
-		}
-
 		pixelsPerUnit := r.FormValue("pixelsPerUnit")
-		px, err := strconv.ParseInt(pixelsPerUnit, 10, 64)
-		if err != nil {
-			glog.Errorf("Error converting string to int64: %v\n", err)
-			return
-		}
-		if px <= 0 {
-			glog.Errorf("pixels per unit must be greater than 0, provided %d\n", px)
-			return
-		}
 
-		var price *big.Rat
-		if pr > 0 {
-			price = big.NewRat(pr, px)
-		}
+		if pricePerUnit != "" && pixelsPerUnit != "" {
+			pr, err := strconv.ParseInt(pricePerUnit, 10, 64)
+			if err != nil {
+				glog.Errorf("Error converting string to int64: %v\n", err)
```

Comment:
Could we return error here like this:
```golang
				msg := fmt.Sprintf("Error converting string to int64: %v\n", err)
				glog.Error(msg)
				w.WriteHeader(http.StatusBadRequest)
				w.Write([]byte(msg))
```
?
And in other places?

Reply:
fixed in 47f9815c